spec/cspec/KernelState_C.thy

```diff
@@ -40,4 +40,9 @@ lemmas conseqPost = HoarePartialDef.conseqPost
 (* Likewise, we'd prefer to get HOL.conj_cong over StateFun.conj_cong *)
 lemmas conj_cong = HOL.conj_cong
 
+(* FIXME isa:  Removes two rules that are too eager for the bitfield proofs.
```

Comment:
FIXME isa

Reply:
This is resolved in a later commit

Reply:
It seems that is not true. Will update (basically just dropping the FIXME part of it, because there is nothing really to fix -- I thought I could do something upstream in Word_Lib, but that doesn't work).